### PR TITLE
Mr/fe migration reduced fe

### PR DESCRIPTION
**Pull request by SeduroDotCom**

### CODE REVIEW

File: src/leihs/inventory/server/routes.clj
Comment (urs-van-binsbergen) [inline src/leihs/inventory/server/routes.clj:1]: removal of lines  73-90 come from a change in the frontend change which is almost certainly not intended?

**Review by urs-van-binsbergen — CHANGES_REQUESTED** [review]
There are still a lot of changes contained which come from the frontend. 
See also comment to src/leihs/inventory/server/routes.clj

**Comment by SeduroDotCom** [comment]
@urs-van-binsbergen @belafonte I tried to reduce modified files as far as possible without breaking the build/test.
If you are really sure which file can be reverted without breaking tests/build then let me know.

File: src/leihs/inventory/server/routes.clj
Comment (SeduroDotCom) [reply src/leihs/inventory/server/routes.clj:1]: Why are you assuming this? Those endpoints are necessary and used to 
1. .. build swagger-ui/swagger.json
2 .. provide /status-endpoint used by cider-tests

**Review by urs-van-binsbergen — APPROVED** [review]
Looks good! Did some quick tests, all good.

**Comment by belafonte** [comment]
Yes looks good to me too!

**Pull request merged**